utils/usbmuxd/files/usbmuxd.hotplug

```diff
@@ -0,0 +1,24 @@
+case "$ACTION" in
+	bind)
+		dev=/sys$DEVPATH
+
+		[ ! -f /tmp/iPhone.lock ] && [ -d ${dev}/net ] &&
+		{
+			readlink ${dev}/driver | grep ipheth &&
+			{
+				sleep 5
+				carrier_path=${dev}/net/*/carrier
+				carrier=`cat ${carrier_path}`
+
+				[ "${carrier}" == "0" ] &&
+				{
+					touch /tmp/iPhone.lock
+					logger -p daemon.error -t iPhone ${carrier_path} = ${carrier}
+					logger -p daemon.error -t iPhone `/usr/bin/usbreset iPhone`
```

Comment:
usbreset is from usbutils. I'm guessing that would be an added dependency.

Reply:
Added. As a side note, when `usbutils` is selected, since it includes `lsusb`, we should disable
Base system, Linux System Utilities, lsusb. Otherwise there is a conflict during install. The same applies for other tools, such as netcat. Can you do this?